README.md

```diff
@@ -55,22 +55,25 @@ If you don't declare any `try` blocks, none of the Scientist machinery is invoke
 The examples above will run, but they're not really *doing* anything. The `try` blocks run every time and none of the results get published. Replace the default experiment implementation to control execution and reporting:
 
 ```ruby
-require "scientist"
+require "scientist/experiment"
 
 class MyExperiment
-  include ActiveModel::Model
   include Scientist::Experiment
 
   attr_accessor :name
 
+  def initialize(name: name)
+    @name = name
+  end
+
   def enabled?
     # see "Ramping up experiments" below
-    super
+    true
   end
 
   def publish(result)
     # see "Publishing results" below
-    super
+    Rails.logger.info result
```

Comment:
@natesholland Yes I can agree and I was concerned with that myself. Any suggestion? Simply `@logger.log result` ?